.circleci/config.yml

```diff
@@ -0,0 +1,77 @@
+version: 2
+jobs:
+  flake8:
+    docker:
+      - image: python:3.7
+    steps:
+      - checkout
+      - run:
+          name: install flake8
+          command: pip install flake8
+      - run:
+          name: test with flake8
+          command: flake8 emissionsapi
+  install buster:
+    docker:
+      - image: debian:buster
+    steps:
+      - checkout
+      - run:
+          name: install on buster
+          command: |
+            apt update
+            apt install -y python3 python3-gdal python3-psycopg2 python3-pycurl \
+                python3-numpy python3-venv
+            python3 -m venv --system-site-packages env
+            ./env/bin/pip install -r requirements.txt
+            cp .testdata/emissionsapi.yml .
+      - run:
+          name: test installation
+          command: |
+              . ./env/bin/activate
+              python -c "import emissionsapi"
+              python -c "import emissionsapi.web"
+              python -c "import emissionsapi.download"
+              python -c "import emissionsapi.db"
+              python -c "import emissionsapi.preprocess"
+  preprocess:
+    docker:
+      - image: debian:buster
```

Comment:
how does the association between container and steps work?

Reply:
See https://circleci.com/docs/2.0/executor-types/#using-multiple-docker-images: 

```
In a multi-image configuration job, all steps are executed in the container created by the first image listed.
```